### PR TITLE
Updated for Hashacks 5.0

### PR DESCRIPTION
**Pull request by arjxn-py**

### CODE REVIEW

File: 2023/index.html [hunk 1]
```diff
@@ -79,7 +79,7 @@ <h1 class="brand">HACKS</h1>
 
                         <section class="content">
                             <!-- <h5>Coming Soon</h5> -->
-                            <h5>Apply for HashHacks 4.0 by October 14th!</h5>
+                            <h5>Apply for HashHacks 5.0 by January 14th!</h5>
```
Comment (Saransh-cpp) [inline 2023/index.html:82]: Uncomment the above line and comment this out for now.

File: 2023/index.html [hunk 2]
```diff
@@ -79,7 +79,7 @@ <h1 class="brand">HACKS</h1>
...
 
                             <a href="https://skillenza.com/challenge/hashhacks4" class="hexagon button" id="register-button">REGISTER</a>
```
Comment (Saransh-cpp) [inline 2023/index.html:84]: Comment for now

File: 2023/index.html [hunk 3]
```diff
@@ -298,11 +298,11 @@ <h5>VP of Data Products at Credit Karma</h5>
                     <li id="sponsors-section">
                         <a class="back-button small blue round button" href="#!"><i class="back"></i></a>
                         <a class="compact-button small round button" href="#sponsors"><i class="sponsors"></i></a>
-                        <a class="primary-button round button" href="#sponsors"><i class="sponsors"></i>SPONSORS</a>
+                        <a class="primary-button round button" href="#sponsors"><i class="sponsors"></i>PREVIOUS SPONSORS</a>
 
                         <div class="above">
                             <section class="content">
-                                If you are interested in sponsoring HashHacks 4.0, <a href="mailto:convoke4.0@gmail.com">email us</a> for more a detailed proposal!
+                                If you are interested in sponsoring HashHacks 5.0, <a href="mailto:convoke5.0@gmail.com">email us</a> for more a detailed proposal!
```
Comment (Saransh-cpp) [inline 2023/index.html]:
```suggestion
                                If you are interested in sponsoring HashHacks 5.0, <a href="mailto:saransh0701@gmail.com">email us</a> for more a detailed proposal!
```

File: 2023/index.html [hunk 4]
```diff
@@ -172,15 +172,15 @@ <h4 class="question">
...
                                 </p>
                             </div>
                             <div>
                                 <h4 class="question">
                                     What if this FAQ didn’t answer my questions?
                                 </h4>
                                 <p class="answer">
-                                    Please email us at <a href="mailto:convoke4.0@gmail.com">convoke4.0@gmail.com</a> if you’re in any way confused or concerned!  We’d love to help you out. You may also reach out to us on Facebook at <a href="https://facebook.com/convoke.ducic" target="_blank">www.facebook.com/convoke.ducic</a> or whatsapp us at 9910268058 or call at 7678640404:)
+                                    Please email us at <a href="mailto:convoke5.0@gmail.com">convoke5.0@gmail.com</a> if you’re in any way confused or concerned!  We’d love to help you out. You may also reach out to us on Facebook at <a href="https://facebook.com/convoke.ducic" target="_blank">www.facebook.com/convoke.ducic</a> or whatsapp us at 9910268058 or call at 7678640404:)
```
Comment (Saransh-cpp) [inline 2023/index.html]:
```suggestion
                                    Please email us at <a href="mailto: saransh0701@gmail.com"> saransh0701@gmail.com</a> if you’re in any way confused or concerned! We’d love to help you out! :)
```

File: 2023/index.html [hunk 5]
```diff
@@ -110,7 +110,7 @@ <h4 class="question">
                                     How does registration work?
                                 </h4>
                                 <p class="answer">
-                                    Registration is open until October 16th, 11:59 pm IST! Applicants will be selected on the basis of Github &amp; LinkedIn profile, resume, past projects and the answers to questions given in the registration form. If you’re accepted, you’ll have two days to confirm your attendance.
+                                    Registration is open until January 16th, 11:59 pm IST! Applicants will be selected on the basis of Github &amp; LinkedIn profile, resume, past projects and the answers to questions given in the registration form. If you’re accepted, you’ll have two days to confirm your attendance.
```
Comment (Saransh-cpp) [inline 2023/index.html]:
```suggestion
                                    Registrations will be opening soon! Applicants will be selected on the basis of Github &amp; LinkedIn profile, resume, past projects, and the answers to questions given in the registration form. If you’re accepted, you’ll have two days to confirm your attendance.
```

File: 2023/index.html [hunk 6]
```diff
@@ -10,17 +10,17 @@
...
-    <meta property="og:title" content="HashHacks 4.0"/>
+    <meta property="og:title" content="HashHacks 5.0"/>
     <meta property="og:type" content="website"/>
     <meta property="og:image" content="http://www.aptusdatalabs.com/wp-content/uploads/2016/04/hackathon-logo.jpg"/>
     <meta property="og:image:width" content="470"/>
     <meta property="og:image:height" content="250"/>
     <meta property="og:image:type" content="image/png"/>
     <meta property="og:url" content="https://hashhacks.tech"/>
-    <meta property="og:description" content="Hack, build, and innovate at Delhi University's best hackathon. Oct 19 - 20."/>
+    <meta property="og:description" content="Hack, build, and innovate at Delhi University's best hackathon. Jan 21 - 22."/>
```
Comment (Saransh-cpp) [inline 2023/index.html]:
```suggestion
    <meta property="og:description" content="Hack, build, and innovate at Delhi University's best hackathon. Jan 28 - 29."/>
```

File: 2023/index.html [hunk 7]
```diff
@@ -39,7 +39,7 @@
 
     <div class="mobile-only mobile-dates">
         <h3 class="light days">12 DAYS REMAINING</h3>
-        <h3>OCTOBER 19-20 2019</h3>
+        <h3>JANUARY 21-22 2023</h3>
```
Comment (Saransh-cpp) [inline 2023/index.html]:
```suggestion
        <h3>JANUARY 28-29 2023</h3>
```

File: 2023/index.html [hunk 8]
```diff
@@ -48,7 +48,7 @@ <h3>OCTOBER 19-20 2019</h3>
         <div class="top">
             <div class="desktop-only lead middle aligned content">
                 <h3 class="light days">12 DAYS REMAINING</h3>
-                <h3>OOCTOBER 19-20 2019</h3>
+                <h3>JANUARY 21-22 2023</h3>
```
Comment (Saransh-cpp) [inline 2023/index.html]:
```suggestion
                <h3>JANUARY 28-29 2023</h3>
```

File: 2023/index.html [hunk 9]
```diff
@@ -172,15 +172,15 @@ <h4 class="question">
                                     What’s the schedule?
                                 </h4>
                                 <p class="answer">
-                                    Check-in starts at 14:30 on 19th Oct followed by introduction at 16:30. Coding period begins at 17:00 and continues till 13:00 the next day. Final judging process will begin at 15:30 and distribution of prizes at 17:00.
+                                    Check-in starts at 09:00 on 21st Jan followed by introduction at 11:00. Coding period begins at 12:00 and continues till 12:00 the next day. Final judging process will begin at 15:30 and distribution of prizes at 17:00.
```
Comment (Saransh-cpp) [inline 2023/index.html]:
```suggestion
                                    Check-in starts at 10:00 on the 28th Jan followed by an introduction at 11:00. Coding period begins at 12:00 and continues till 12:00 the next day. The final judging process will begin at 15:30 and the distribution of prizes at 17:00.
```

File: 2023/stylesheets/app.css [hunk 10]
```diff
@@ -82,7 +82,7 @@ button, .button {
       color: #252C37;
       background-image: url("../assets/images/hex-button-border-disabled@2x.png"); }
   button.round, .button.round {
-    min-width: 180px;
+    min-width: 250px;
```
Comment (Saransh-cpp) [inline 2023/stylesheets/app.css:85]: This was for?

**Review by Saransh-cpp — APPROVED** [review]
Thank you, @arjxn-py! Very minor comments below. I will resolve the maximum but some of them will have to be resolved by you.

File: 2023/stylesheets/app.css (hunk 10 above)
Comment (arjxn-py) [reply 2023/stylesheets/app.css:85]: This is to preserve styling @Saransh-cpp

File: 2023/index.html (hunk 1 above)
Comment (arjxn-py) [reply 2023/index.html:82]: It has been done Thank You.

File: 2023/index.html (hunk 2 above)
Comment (arjxn-py) [reply 2023/index.html:84]: Done hence marking resolved.

**Pull request merged**